src/Paket/Commands.fs

```diff
@@ -700,7 +700,7 @@ with
             | Version -> "show Paket version"
             | From_Bootstrapper -> "call coming from the '--run' feature of the bootstrapper"
 
-let commandParser = ArgumentParser.Create<Command>(programName = "paket", errorHandler = new ProcessExiter())
+let commandParser = ArgumentParser.Create<Command>(programName = "paket", errorHandler = new ProcessExiter(), checkStructure = false)
```

Comment:
Just out of curiosity what is the impact of this? Is there an impact?

Reply:
On performance it's faster :)

It also doesn't check the argument DU for incoherences, but it's forced in unit tests so it would fail there.